src/wrapped/wrappedlibc_private.h

```diff
@@ -1292,7 +1292,7 @@ GO(posix_spawn, iFpppppp)
 // posix_spawnattr_setschedpolicy
 // posix_spawnattr_setsigdefault
 // posix_spawnattr_setsigmask
-// posix_spawn_file_actions_addclose
+GO(posix_spawn_file_actions_addclose, iFpipii)
```

Comment:
Look here: https://linux.die.net/man/3/posix_spawn_file_actions_addclose signature is justy `iFpi`